server/src/main/java/org/elasticsearch/action/search/SearchTransportAPMMetrics.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.search;
+
+import org.elasticsearch.telemetry.TelemetryProvider;
+import org.elasticsearch.telemetry.metric.LongHistogram;
+import org.elasticsearch.telemetry.metric.MeterRegistry;
+
+public class SearchTransportAPMMetrics {
+    private final LongHistogram actionLatencies;
+
+    public SearchTransportAPMMetrics(MeterRegistry meterRegistry) {
+        this(
+            meterRegistry.registerLongHistogram(
+                "es.transport.action.latency.histogram",
```

Comment:
The class is about search but the metric does not include search, is that something that we should fix?

Reply:
Good point.  I'm thinking we'll later add another metric at the coordinator,  like so:
`es.search.phase.coordinator.latency.histogram`